dialyzer.ignore-warnings

```diff
@@ -1,3 +1,6 @@
 apps/omg_performance/
 apps/omg/test
 apps/omg_eth/test
+
+Function domain_separator/5 has no local return
```

Comment:
ah ah :) these need to be addressed not ignored :)

Reply:
Agree, don't want to fight with it right now. Seems it wrongly deffer a type from ABI... Could you please take a look?

Reply:
Yep. 
```
{:uint, integer()}
          | :bool
          | :bytes
          | :string
          | :address
          | {:array, type}
          | {:array, type, non_neg_integer}
          | {:tuple, [type]}
```
`[{:bytes, 32}]` are not allowed.

Reply:
It just says so, but don't trust it :) https://github.com/poanetwork/ex_abi/blob/cf58a07b0b838452ba253d04e61788ef7aca4dc1/lib/abi/type_encoder.ex#L200
ABI doesn't provide specs, so I guess this is derived by dialyzer and it's picking on me (Why's Everybody Always Pickin' On Me? :smile: )